source/includes/authentication.cpp

```diff
@@ -1,72 +1,48 @@
 // start-scram-sha-256
-#include <mongocxx/client.hpp>
-#include <mongocxx/uri.hpp>
-
 auto uri = mongocxx::uri("mongodb://<username>:<password>@<hostname>:<port>/?"
                          "authSource=admin&authMechanism=SCRAM-SHA-256");
-mongocxx::client client(uri);
+auto client = mongocxx::client client(uri);
```

Comment:
```suggestion
auto client = mongocxx::client(uri);
```
Incorrect repetition of variable name in declaration.

Also applies to other instances in this file.